src/apps/chatter/hooks/use-room-password.js

```diff
@@ -26,7 +26,9 @@ export function useRoomPassword (roomId, password) {
     socketRef.current = new window.WebSocket(socketUrl)
     socketRef.current.onopen = () => {
       console.log('WebSocket open to url ', socketUrl)
-      checkPassword()
+      if (password !== '') {
```

Comment:
Does useRoom need to be broken up? Or is useRoomPassword going to replace useRoom?

Reply:
useRoom was beginning to feel clunky with all the returned items, but now that I've shortened it down I can try to put them back into one again.

Reply:
This is perhaps not the best name for this hook. Its real purpose is to check the validity of a password in order to know when the password prompt should disappear, revealing the chatbox. Before this wall all in one hook, but led to awkwardness such as useless sent messages, possible race conditions, etc.